src/Control/Effect/Sum.hs

```diff
@@ -1,14 +1,16 @@
-{-# LANGUAGE ConstraintKinds, DeriveGeneric, DeriveTraversable, FlexibleContexts, FlexibleInstances, KindSignatures, MultiParamTypeClasses, TypeOperators, UndecidableInstances #-}
+{-# LANGUAGE DeriveGeneric, DeriveTraversable, FlexibleContexts, FlexibleInstances, KindSignatures, MultiParamTypeClasses, TypeOperators, UndecidableInstances #-}
+-- | Operations on /sums/, combining effects into a /signature/.
 module Control.Effect.Sum
-( (:+:)(..)
-, Member
-, Inject(..)
-, Project(..)
+( -- * Membership
+  Member(..)
+  -- * Sums
+, (:+:)(..)
```

Comment:
The docs read a lot better with `Member` listed first, due to the extremely large number of instances involving `:+:`.